### ISSUE TITLE
MZmine 2.53 software interface has overlapping images

### ISSUE THREAD

**Issue by Zhengyu1014** [mzmine/mzmine2#909]
### Basic information

* My operating system: Win10
* My MZmine version: 2.53

### What happened

when I useMZmine, there are overlapping images in the GUI interface. I started to think it might be the Java version. However, I have re installed Java and configured environment variables in my computer, which does not solve this problem. And I tried to re install the computer system to solve the problem mentioned above, but the same problem still appeared after re installing the system.

![1618797977(1)](https://user-images.githubusercontent.com/82654833/115172794-93fb8500-a0f8-11eb-991f-6d677ad5f554.jpg)
![1618798090(1)](https://user-images.githubusercontent.com/82654833/115172806-95c54880-a0f8-11eb-9d18-b26e5b80a087.jpg)
![1618798120(1)](https://user-images.githubusercontent.com/82654833/115172808-96f67580-a0f8-11eb-88af-3defb31f3b4e.jpg)
![1618798180(1)](https://user-images.githubusercontent.com/82654833/115172812-98c03900-a0f8-11eb-82c0-99e21d2bf8e8.jpg)

**Comment by SteffenHeu** [comment]
I've never seen that before. Have you had the issue with other Java applications, too?
Does it appear every time? Does it work on a different PC?

**Comment by Zhengyu1014** [comment]
Yes, other Java applications have the same situation, such as SIRIUS（https://bio.informatik.uni-jena.de/software/sirius/）and Cytoscape(https://cytoscape.org/).The same thing happens every time, sometimes after importing data. This is not work on a different PC.
